### ISSUE TITLE
Select widget sync with filters

### ISSUE THREAD

**Issue by Maaark** [algolia/instantsearch.js#1115]
Hello,

I'm using this widget https://github.com/algolia/instantsearch.js/issues/1022 and I would like to have it synced with the other filters. 

So if I apply a filter (for example price) and a option doesn't match the shown products it won't be displayed in the select?

How can I do this?

**Comment by vvo** [comment]
Can you provide a jsFiddle to demo it and so we can work from here?

**Comment by Maaark** [comment]
Never mind, managed to fix it myself. But I do have some other things I can't figure out, I see that there is a email api support but I can't find the email address?

**Comment by vvo** [comment]
I am not sure to get your last question, can you rephrase?

**Comment by Maaark** [comment]
Haha It was quite vague indeed.. I meant to ask for the 'API support' email address, but I've already found it.

**Comment by vvo** [comment]
Great! Thanks for using Algolia and instantsearch.js
